cpp-linux/yum/arrow.spec.in

```diff
@@ -60,13 +60,17 @@ mkdir cpp/build
 cd cpp/build
 %cmake3 .. \
   -DCMAKE_BUILD_TYPE=$build_type \
+  -DARROW_ORC=ON \
 %if %{use_python}
-  -DARROW_PYTHON=yes \
+  -DARROW_PYTHON=ON \
+  -DPythonInterp_FIND_VERSION=ON \
+  -DPythonInterp_FIND_VERSION_MAJOR=3 \
 %endif
 %if %{_centos_ver} == 6
-  -DARROW_BOOST_VENDORED=yes \
+  -DARROW_BOOST_VENDORED=ON \
 %endif
```

Comment:
Do we need an `%else` here to say `-DARROW_BOOST_USE_SHARED=OFF`?

Reply:
No we don't. `ARROW_BOOST_USE_SHARED` is `ON` by default.

Reply:
Ah, of course. Thanks.